src/AddThreepid.js

```diff
@@ -226,19 +260,39 @@ export default class AddThreepid {
 
         if (await MatrixClientPeg.get().doesServerSupportSeparateAddAndBind()) {
             if (this.bind) {
+                const identityServerDomain = MatrixClientPeg.get().idBaseUrl.split("://")[1];
```

Comment:
I think this is left over from merging develop? This shouldn't be needed now, as `getIdServerDomain()` a few lines below takes care of it.